source/Camera/InfoUtilOther/ZividBenchmark/ZividBenchmark.cpp

```diff
@@ -525,6 +585,8 @@ int main()
         benchmarkCapture2D(camera, makeSettings2D(exposureTime), numFrames2D);
         printHeader("TEST 7: Save");
         benchmarkSave(camera, numFramesSave);
+        printHeader("TEST 8: CopyData");
+        benchmarkCopyDataTime(camera);
```

Comment:
Move this before benchmarkCapture2D because it is a more natural flow of tests. First, connect/disconnect, then all 3D captures, then all 3D copy commands, then 2D capture, then saving stuff.

Reply:
Ok, after talking a bit with the software team, I think we should actually do the following:

Move

```
printHeader("TEST 8: CopyData");
benchmarkCopyDataTime(camera);
```

after `benchmarkCapture2D`. Then, add in `benchmarkCapture2D ` one 2D capture, basically this:

`const auto frame2D = camera.capture(settings2D);`

and benchmark getting 2D image function call (in a loop), basically this:

`const auto image = frame2D.imageRGBA();`